yucca/pipeline/managers/YuccaManager.py

```diff
@@ -163,7 +163,7 @@ def initialize(
 
         seed_config = seed_everything_and_get_seed_config(ckpt_seed=self.ckpt_config.ckpt_seed)
 
-        plan_config = self.get_plan_config(
```

Comment:
This should be reverted, as this allows other managers to intervene how they get the plan config. if e.g. they want to manually specify the PlanConfig, without having to overwrite the whole initialize method

Reply:
I dont think it makes sense to challenge this, but just want to state that i find that extremely bad practice.